neqo-crypto/src/lib.rs

```diff
@@ -17,15 +17,15 @@ mod agentio;
 mod cert;
 pub mod constants;
 mod convert;
-mod err;
+pub mod err;
```

Comment:
I missed these previously, but I think that these are not good changes.  These were internal for a reason and making them public makes a lot more public than I would like.  It exposes all of the SSL FFI for instance.

Can you `pub use` just the types that you specifically want to expose here instead?

Reply:
done.